### PR TITLE
(feat) O3-2258: Put RDE behind a feature flag

### PR DESCRIPTION
**Pull request by brandones**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
Puts RDE functionality behind a feature flag.

## Screenshots
Screencast from 2023-07-18 14-41-12

## Related Issue

https://issues.openmrs.org/browse/O3-2258

### CODE REVIEW

**Review by ibacher — APPROVED** [review]
If you update to core -pre.905 (now available),  we shouldn't need to use the internal import.

**Comment by brandones** [comment]
Good catch, thanks @ibacher !

File: packages/esm-patient-chart-app/src/actions-buttons/mark-patient-alive.component.tsx [hunk 1]
```diff
@@ -6,7 +6,6 @@ import { usePatientDeceased } from '../deceased/deceased.resource';
 
 interface MarkPatientAliveOverflowMenuItemProps {
   patientUuid?: string;
-  launchPatientChart?: boolean;
```
Comment (brandones) [inline packages/esm-patient-chart-app/src/actions-buttons/mark-patient-alive.component.tsx:9]: cruft

**Review by denniskigen — APPROVED** [review]

**Comment by denniskigen** [comment]
Much appreciated, @brandones!

**Pull request merged**